src/api/routes.ts

```diff
@@ -17,6 +17,7 @@ export const buildDeleteItemsRoute = (ids: UUID[]) =>
 export const buildGetChildrenRoute = (id: UUID) =>
   `${ITEMS_ROUTE}/${id}/children`;
 export const buildGetItemRoute = (id: UUID) => `${ITEMS_ROUTE}/${id}`;
+export const buildGetItemsRoute = (ids: UUID[]) => `${ITEMS_ROUTE}?${ids.map(i => "id="+i).join("&")}`;
```

Comment:
I'm not always using it, but there is this `qs` library which is really useful for querystrings. Please have a look https://www.npmjs.com/package/qs